rekuho/src/lib/LinesView.svelte

```diff
@@ -0,0 +1,63 @@
+<!--
+    LinesView contains lines of text
+-->
```

Comment:
So..... why do we have this?

Reply:
So that we can write more documentation later.